realize/cli_test.go

```diff
@@ -11,7 +11,7 @@ import (
 
 func TestRealize_Stop(t *testing.T) {
 	r := Realize{}
-	r.Projects = append(r.Schema.Projects, Project{exit: make(chan os.Signal, 1)})
+	r.Projects = []Project{exit: make(chan os.Signal, 1)}
```

Comment:
This is actually a syntax error now. You're building an slice but only have one pair of brackets. It should be:
```go
 	r.Projects = []Project{{exit: make(chan os.Signal, 1)}}
```